### DISCUSSION TITLE
eGPU Enclosure over USB-C (Lightning 3)

### DISCUSSION THREAD

**Discussion by beadon** [NVIDIA/open-gpu-kernel-modules#449] [Q&A]
I am tracking an issue with some type of crashing interaction of the Linux kernel, NVIDIA proprietary drivers and Gnome or Xorg exposed over a Lightning 4 port running at Lightning 3 connection speeds to a NViDIA GeForce 1650 connected to multiple display ports. ( sitting inside a Razer Chroma enclosure )

Has anyone put this kind of configuration together ?  I seem to be running into serious showstoppers!

**Comment by sTiKyt** [comment]
Do you have the same issue with open driver from this repo?

**Comment by beadon** [comment]
I will give it a try.
